.gitmodules

```diff
@@ -0,0 +1,12 @@
+[submodule "AviSynthPlus"]
+	path = AviSynthPlus
+	url = https://github.com/AviSynth/AviSynthPlus.git
+[submodule "libp2p"]
+	path = libp2p
+	url = https://github.com/sekrit-twc/libp2p.git
+[submodule "vsrepo"]
+	path = vsrepo
+	url = https://github.com/vapoursynth/vsrepo.git
+[submodule "zimg"]
+	path = zimg
+	url = https://github.com/sekrit-twc/zimg.git
```

Comment:
Iirc, Myr used this once, and was dissatisfied.
I don't like the feature either. You're gonna have to find another solution for that.

Reply:
This isn't a solution to anything, the build scripts grab em from elsewhere, it's just that i think it's just nornal to link to other repos in github too.
If myrsloik doesn't like it for w/e reason we can just delete the file